azure_functions_worker/bindings/meta.py

```diff
@@ -184,3 +215,57 @@ def to_outgoing_param_binding(binding: str, obj: typing.Any, *,
         return protos.ParameterBinding(
             name=out_name,
             data=rpc_val)
+
+
+def get_deferred_binding(bind_name: str,
+                         pytype: typing.Optional[type] = None) -> object:
+    binding = None
+
+    # Checks if pytype is a supported sdk type
+    if (DEFERRED_BINDINGS_REGISTRY is not None
+            and DEFERRED_BINDINGS_REGISTRY.check_supported_type(pytype)):
+        # Returns deferred binding converter
+        binding = DEFERRED_BINDINGS_REGISTRY.get(bind_name)
+
+    # This will return None if not a supported type
+    return binding
+
+
+def deferred_bindings_decode(binding: typing.Any,
+                             pb: protos.ParameterBinding, *,
+                             pytype: typing.Optional[type],
+                             datum: typing.Any,
+                             metadata: typing.Any):
+    # This cache holds deferred binding types (ie. BlobClient, ContainerClient)
+    # That have already been created, so that the worker can reuse the
+    # Previously created type without creating a new one. It allows these
+    # Types to be singleton.
+    global DEFERRED_BINDINGS_CACHE
+
+    # Check if the type is already in the cache
+    # If dict is empty or key doesn't exist, deferred_binding_type is None
+    deferred_binding_type = DEFERRED_BINDINGS_CACHE.get((pb.name, pytype,
+                                                         datum.value.content),
+                                                        None)
+
+    # If the type is in the cache, return it
+    if deferred_binding_type is not None:
+        return deferred_binding_type
+    # Otherwise, create the specified type and add it to the cache
+    else:
+        deferred_binding_type = binding.decode(datum, trigger_metadata=metadata,
+                                               pytype=pytype)
+        DEFERRED_BINDINGS_CACHE[(pb.name, pytype, datum.value.content)]\
+            = deferred_binding_type
+        return deferred_binding_type
+
+
+def set_deferred_bindings_flag(param_anno: type):
```

Comment:
What is the purpose of this method? is it to set the flag in raw_bindings or just to tell if the featue is enabled?